docs/Deploy/deploy-fleet.md

```diff
@@ -138,7 +139,10 @@ This workflow takes about 30 minutes to complete and supports between 10 and 350
     terraform apply
     ```
 
-12. That’s it! You should now be able to log in to Fleet and [enroll a host](https://fleetdm.com/docs/using-fleet/enroll-hosts).
+12. That’s it! Next, you can skip ahead and already log in to Fleet to [enroll a host](https://fleetdm.com/docs/using-fleet/enroll-hosts), or you can continue with our other Deployment documents for other options that might fit your environment:
+- [Single sign-on (SSO)](https://fleetdm.com/docs/deploy/single-sign-on-sso)
+- [Upgrading Fleet](https://fleetdm.com/docs/deploy/upgrading-fleet)
+- [Reference architectures](https://fleetdm.com/docs/deploy/reference-architectures)
```

Comment:
Let's revert this change. We already have all these links in the deployment section of the side nav, and introducing duplication here makes it harder to keep these up-to-date.
```suggestion
12. That’s it! You should now be able to log in to Fleet and [enroll a host](https://fleetdm.com/docs/using-fleet/enroll-hosts).
```